CONTRIBUTING.md

```diff
@@ -61,6 +59,24 @@ TODO: Code Style Guide
 
 ### Issues
 
+1. Check for Existing Issues
```

Comment:
I like this text a lot 🙌  We should definitely add some of this content to our templates in repo-scaffolder